### DISCUSSION TITLE
Is it possible to run application using CefSharp in Linux with wine?

### DISCUSSION THREAD

**Discussion by diman4ik** [cefsharp/CefSharp#4219] [General]
I tried to run Winforms application using CefSharp in Linux using wine and mono. I get System.TypeLoadException because CefSharp.Core.Runtime can't be loaded. Is it possible to run CefSharp application with wine?

**Comment by amaitland** [comment]
I've never tried, so no idea. `CefSharp.Core.Runtime.dll` requires `Visual C++ 2019` or greater, you'd likely still need to include the `VC++ Runtime` somehow.

**Comment by jiiiijiij** [comment]
> I tried to run Winforms application using CefSharp in Linux using wine and mono. I get System.TypeLoadException because CefSharp.Core.Runtime can't be loaded. Is it possible to run CefSharp application with wine?

Had you able to make it work ?

**Reply by trivalik** [reply]
I think the best solution would be to create something like https://github.com/OutSystems/cef.redist.linux for CefSharp, this would allow to run it natively.

**Reply by amaitland** [reply]
That won't change the fact that `CefSharp` is build using `Visual C++` which is `Windows Only`

**Reply by trivalik** [reply]
I know.

Is there a reason why this C++ Wrapper/Layer is there? Since CefGlue does not having this.
